recipe/meta.yaml

```diff
@@ -27,9 +27,10 @@ requirements:
 test:
   imports:
     - pdfminer
-
+  source_files:
+    - tools/pdf2txt.py
   commands:
-    - pdf2txt.py -h
+    - python tools/pdf2txt.py -h
```

Comment:
Why this change? Isn't this supposed to be installed by the package?

Reply:
It was my misunderstanding. But `pdf2txt.py -h` leads build failure because [`pdf2txt.py` doesn't have a shebang line](https://github.com/pdfminer/pdfminer.six/blob/20200402/tools/pdf2txt.py) as of v20200402.
This problem is fixed by [the latest commit](https://github.com/pdfminer/pdfminer.six/commit/7eff108fa5b4378c150c3ac1487e60b627a191e2). So I'll revert and comment out this line, and I'll uncomment after the next release.

Reply:
Right. So this failing means the package itself would be broken. The test here makes sure the package works when installed so we cannot change it like this.

Reply:
Do you mean I should not comment out `pdf2txt.py -h`?

Reply:
Yup. We have to wait until a working version is shipped.

Reply:
I got it. Thx.